support/user-guide/subaccounts.md

```diff
@@ -49,7 +50,8 @@ In addition, your subaccount users will be able to:
 * Get raw message event data via the API
 * View and manage the suppression list via the API​
 * Create their own webhook via the API
-* Create, edit, view, and use templates
+* Create, edit, view, and use templates via the API
+* Access analytics and event data, including Signals, via the UI
```

Comment:
@haleysolomon shouldn't this be just "Access Signals analytics through the UI"?

Reply:
I was a little hesitant to assume everyone new what that was. I changed the wording and linked to a doc with more info, that make me feel better :-D